ui/app/components/inference/TryWithButton.tsx

```diff
@@ -9,25 +9,27 @@ import {
 import { ChevronDown } from "lucide-react";
 import { Compare } from "~/components/icons/Icons";
 
-export interface TryWithVariantButtonProps {
-  variants: string[];
-  onVariantSelect: (variant: string) => void;
+export interface TryWithButtonProps {
+  options: string[];
+  onOptionSelect: (variant: string) => void;
   isLoading: boolean;
+  model?: boolean;
 }
 
-export function TryWithVariantButton({
-  variants,
-  onVariantSelect,
+export function TryWithButton({
+  options: variants,
+  onOptionSelect: onVariantSelect,
   isLoading,
-}: TryWithVariantButtonProps) {
+  model,
```

Comment:
Using a boolean for now - we could switch this to a string in the future if there are more than the two states

Reply:
You've got it!